terminal.md

```diff
@@ -0,0 +1,39 @@
+5. Get out of a directory :
+
+-If I'm in the work directory, what do I need to type to get to to_do?  
+answer: cd ..
+
+-If I'm in the projects_to_delegate directory, what two commands do I need to type to get to to_do?  
+answer: 1- cd ..
+        2- cd ..
+
+-I'm in the home directory. What three commands do I need to type to get to projects_to_delegate?
+answer: 1- cd ... 
+        2- cd work
+        3- cd projects_to_delegate
+
+I'm in the projects_to_delegate directory. What three commands do I need to type to get to home?
+answer: 1-cd ..
+        2-cd ..
+        3-cd home 
+
+
+10.  Remove a Directory and Its Contents
+
+-I'm in to_do. What do I type to remove random.txt?
+answer: rm  random.txt
+
+-I'm in to_do. What do I type to remove the home directory?
+answer: rm -rf home
+
+-I'm in the work directory. What two commands do I type to remove the home directory?
+answer: 1- cd .. 
+        2- rm -rf home
+
+-I'm in the projects_to_delegate directory. What two commands do I type to remove the directory I'm currently in?
+answer: 1-cd ..
+        2- rm -rf  projects_to_delegate
+
+-I'm in projects_to_delegate. What four commands do I need to type to remove the cleaning.txt file?
+answer: 1- cd ../../home
+        2- rm cleaning.txt
```

Comment:
Looks good to me.

Just a tip: `cd ..` twice can also be done like this `cd ../..` or just `../..`.